test/test_functional_tensor.py

```diff
@@ -795,6 +795,32 @@ def test_solarize2(device, dtype, config, channels):
     )
 
 
+@pytest.mark.parametrize("config", [{"threshold": threshold} for threshold in [0.0, 0.25, 0.5, 0.75, 1.0,]])
+def test_solarize_threshold1_bound(config):
+    img = torch.rand((3, 12, 23)).to("cpu")
```

Comment:
It doesn't run on GPU by default:
```suggestion
    img = torch.rand((3, 12, 23))
```

Reply:
Thanks! Let me correct it to run on GPU as well.

Reply:
I should have been clearer here. Your test doesn't benefit greatly from running both on CPU and GPU because we just check if the test passes/fails due to the provided threshold. So here I was just suggesting to remove the `.to("cpu")` and keep everything else as-is. But given that all other tests run in both CPU/GPU and your new tests are super fast, there is no harm in running them in both. :)